index.html

```diff
@@ -59,15 +60,24 @@ <h1 class="narson">I'm Narson.
                 </a>
             </li>
         </ul>
+    </div>
 
     </section>
 
     <section class="work">
         <div class="card-container">
-            <img class="sary1" src="media/Snapshoot Portfolio2.svg" alt="Project Snapshoot" />
+            <img class="sary" src="media/Snapshoot-Portfolio-desktop.svg" alt="Project Snapshoot" />
+            
+            <div class="work-fonony">
+
             <h2 class="card-tittle">Tonic</h2>
-            <div class="frame2-img">
-                <img src="media/Frame 2.svg" />
+            <div class="frame3-img">
+                <!-- <img src="media/Frame 2.svg" /> -->
+                <ul class="project-info">
+                    <li class="fb">CANOPY</li>
+                    <li class="full">Back End Dev</li>
+                    <li class="year">2015</li>
+                </ul>
             </div>
```

Comment:
- [ ] **[OPTIONAL]** Please, make sure your work card is responsive and stick to the figma design even when resized.
**Your project work card when resized**
![image](https://user-images.githubusercontent.com/74411135/230723002-04b8faba-9836-47b9-849c-23f358d8a610.png)

This will make your project way prettier which will improve the user experience 📱  😄 👍 
**Note:** This is a project requirement potentially missed by the previous reviewer. So please consider to implement it.

Reply:
- [ ] Nice effort making the changes :wink:  However your project description texts are not aligned. Kindly check it 

### Your Design 
![Screenshot from 2023-04-08 19-02-00](https://user-images.githubusercontent.com/91142432/230738657-8926e1ff-5977-4d96-843c-583aa03a2473.png)

### Figma Design
![Screenshot from 2023-04-08 19-04-00](https://user-images.githubusercontent.com/91142432/230738686-5dfcd971-4c73-47c6-b929-90ce3b0af5aa.png)